nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -834,6 +834,31 @@ CREATE ROLE postgres LOGIN SUPERUSER;
      functionally redundent.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     The package <package>nextcloud17</package> has been removed and <package>nextcloud18</package> was marked as insecure
+     since both of them will <link xlink:href="https://docs.nextcloud.com/server/19/admin_manual/release_schedule.html">get out
+     of maintenance</link> within the lifetime of 20.09.
```

Comment:
how about `will be EOL (end of life) within the timeline of 20.09`?

Reply:
Even just `s/get/be/` would make this sound less awkward.